acceptance/tests/resource/package/common_package_name_in_different_providers.rb

```diff
@@ -41,6 +41,12 @@ def verify_absent(hosts, pkg)
   clean_rpm agent, rpm_options
   setup_rpm agent, rpm_options
   send_rpm agent, rpm_options
+
+  # DNF requires a cache reset to make local repositories accessible.
+  cmd = rpm_provider(agent)
+  if cmd == 'dnf'
+    on(agent, 'dnf clean metadata')
+  end
```

Comment:
@joshcooper I believe this was an issue specifically because of the local repo being created for the test, though I'll manually see what DNF does when a regular repo is added.

Yeah, I think that logic could make sense in the `send_rpm` method of `RpmUtils`. I'll move it over.